### DISCUSSION TITLE
How to disable elements from being edited on the page?

### DISCUSSION THREAD

**Discussion by advancedsoftwarecanada** [GrapesJS/grapesjs#5744] [Q&A]
Can anyone suggest?

![image](https://github.com/GrapesJS/grapesjs/assets/1661561/3e7f43bd-bdfb-4a7f-ad39-de7a11a4ddf5)

**Comment by artf** [comment]
Try to check the [component properties](https://grapesjs.com/docs/api/component.html#properties)
